sipa/blueprints/usersuite.py

```diff
@@ -673,3 +677,92 @@ def get_apple_wlan_mobileconfig():
         as_attachment=True,
         download_name="agdsn.mobileconfig",
     )
+
+
+@bp_usersuite.route("/request-repayment", methods=["GET", "POST"])
+@login_required
+def request_repayment():
+    """
+    Request a repayment of excess membership contributions
+    """
+    # Check whether there is a repayment request already
+    if current_user.get_request_repayment():
+        flash(gettext("Du hast bereits einen Rücküberweisungsantrag abgesendet, der sich noch in Bearbeitung befindet."), "warning")
+        return redirect(url_for(".index"))
+
+    form = RequestRepaymentForm()
+
+    if form.validate_on_submit():
+        return redirect(
+            url_for(
+                ".request_repayment_confirm",
+                beneficiary=form.beneficiary.data,
+                iban=form.iban.data,
+                amount=form.amount.data,
+            )
+        )
+    elif form.is_submitted():
+        flash_formerrors(form)
+
+    form_args = {
+        "form": form,
+        "cancel_to": url_for(".index"),
+        "submit_text": gettext("Weiter"),
+    }
+
+    return render_template(
+        "generic_form.html",
+        page_title=gettext("Rücküberweisung beantragen"),
+        form_args=form_args,
+    )
+
+
+@bp_usersuite.route("/request-repayment/confirm", methods=["GET", "POST"])
+@login_required
+def request_repayment_confirm():
+    """
+    Request a repayment of excess membership contributions
+    """
+    # Check whether there is a repayment request already
+    if current_user.get_request_repayment():
+        flash(gettext("Du hast bereits einen Rücküberweisungsantrag abgesendet, der sich noch in Bearbeitung befindet."), "warning")
+        return redirect(url_for(".index"))
+
+    beneficiary = request.args.get("beneficiary", None)
+    iban = request.args.get("iban", None, partial(IBAN, validate_bban=True))
+    amount = request.args.get("amount", None, Decimal)
+
+    form = RequestRepaymentConfirmForm()
+
+    if None not in (beneficiary, iban, amount):
+        balance = current_user.finance_information.balance.raw_value
+
+        form.beneficiary.data = beneficiary
+        form.iban.data = iban.formatted
+        form.bic.data = iban.bic.formatted
+        form.bank.data = iban.bank_name
+        form.amount.data = amount
+        form.estimated_balance.data = balance - amount
+    else:
+        return redirect(url_for(".request_repayment"))
```

Comment:
Flip this and use an early return to avoid the indentation layer:
```suggestion
    if None in (beneficiary, iban, amount):
        return redirect(url_for(".request_repayment"))

    balance = current_user.finance_information.balance.raw_value
    form.beneficiary.data = beneficiary
    form.iban.data = iban.formatted
    form.bic.data = iban.bic.formatted
    form.bank.data = iban.bank_name
    form.amount.data = amount
    form.estimated_balance.data = balance - amount
```